api_tests/files/views/test_file_detail.py

```diff
@@ -27,449 +27,467 @@ def _dt_to_iso8601(value):
-        res = self.app.get(url, auth=self.user.auth, expect_errors=True)
-        assert_equal(res.status_code, 404)
+        # test_get_file_non_file_guid_gives_404
+        url = '/{}files/{}/'.format(API_BASE, node._id)
+        res = app.get(url, auth=user.auth, expect_errors=True)
+        assert res.status_code == 404
 
-    def test_current_version_is_equal_to_length_of_history(self):
-        res = self.app.get(self.file_url, auth=self.user.auth)
-        assert_equal(res.json['data']['attributes']['current_version'], 1)
+    def test_current_version_is_equal_to_length_of_history(self, app, user, file_url, file):
+        res = app.get(file_url, auth=user.auth)
+        assert res.json['data']['attributes']['current_version'] == 1
         for version in range(2, 4):
-            self.file.create_version(self.user, {
+            file.create_version(user, {
                 'object': '06d80e' + str(version),
                 'service': 'cloud',
                 osfstorage_settings.WATERBUTLER_RESOURCE: 'osf',
             }, {'size': 1337,
                 'contentType': 'img/png'
             }).save()
-            res = self.app.get(self.file_url, auth=self.user.auth)
-            assert_equal(res.json['data']['attributes']['current_version'], version)
+            res = app.get(file_url, auth=user.auth)
+            assert res.json['data']['attributes']['current_version'] == version
 
     # Regression test for OSF-7758
-    def test_folder_files_relationships_contains_guid_not_id(self):
-        self.folder = self.node.get_addon('osfstorage').get_root().append_folder('I\'d be a teacher!!')
-        self.folder.save()
-        self.folder_url = '/{}files/{}/'.format(API_BASE, self.folder._id)
-        res = self.app.get(self.folder_url, auth=self.user.auth)
+    def test_folder_files_relationships_contains_guid_not_id(self, app, user, node):
+        folder = node.get_addon('osfstorage').get_root().append_folder('I\'d be a teacher!!')
+        folder.save()
+        folder_url = '/{}files/{}/'.format(API_BASE, folder._id)
+        res = app.get(folder_url, auth=user.auth)
         split_href = res.json['data']['relationships']['files']['links']['related']['href'].split('/')
-        assert_in(self.node._id, split_href)
-        assert_not_in(self.node.id, split_href)
+        assert node._id in split_href
+        assert node.id not in split_href
+
+@pytest.mark.django_db
+class TestFileVersionView:
 
-class TestFileVersionView(ApiTestCase):
-    def setUp(self):
-        super(TestFileVersionView, self).setUp()
+    @pytest.fixture()
+    def node(self, user):
+        return ProjectFactory(creator=user)
 
-        self.user = AuthUserFactory()
-        self.node = ProjectFactory(creator=self.user)
+    @pytest.fixture()
+    def osfstorage(self, node):
+        return node.get_addon('osfstorage')
 
-        self.osfstorage = self.node.get_addon('osfstorage')
+    @pytest.fixture()
```

Comment:
Not sure about this, but root_node appears to only be used in the file fixture, so it could be moved into that fixture as it isn't entirely necessary